src/libopensc/card-openpgp.c

```diff
@@ -3302,36 +3300,36 @@ pgp_erase_card(sc_card_t *card)
 /**
  * ABI: ISO 7816-9 CARD CTL - perform special card-specific operations.
  */
-static int
+	static int
 pgp_card_ctl(sc_card_t *card, unsigned long cmd, void *ptr)
 {
 	int r;
 
 	LOG_FUNC_CALLED(card->ctx);
 
 	switch(cmd) {
-	case SC_CARDCTL_GET_SERIALNR:
-		memmove((sc_serial_number_t *) ptr, &card->serialnr, sizeof(card->serialnr));
-		LOG_FUNC_RETURN(card->ctx, SC_SUCCESS);
-		break;
-	case SC_CARDCTL_OPENPGP_SELECT_DATA:
-		pgp_select_data(card, *((u8 *) ptr));
-		LOG_FUNC_RETURN(card->ctx, SC_SUCCESS);
-		break;
+		case SC_CARDCTL_GET_SERIALNR:
+			memmove((sc_serial_number_t *) ptr, &card->serialnr, sizeof(card->serialnr));
+			LOG_FUNC_RETURN(card->ctx, SC_SUCCESS);
+			break;
+		case SC_CARDCTL_OPENPGP_SELECT_DATA:
+			r = pgp_select_data(card, *((u8 *) ptr));
+			LOG_FUNC_RETURN(card->ctx, r);
+			break;
```

Comment:
I do not remember I suggested indenting the `case` blocks inside the switches. I think the current coding guidelines should allow both, but less indent is always better in my opinion. But I still did not manage to get the final `clang-format` in (see #1641).
Reformatting the whole blocks with other changes in single commit makes it hard to gasp what you actually changed.

Reply:
case in same indent as switch, check